tests/test_uuid_utilities.py

```diff
@@ -0,0 +1,46 @@
+import mock
+
+from django.test import TestCase
+from morango.utils.uuids import DatabaseIDModel, InstanceIDModel, UUIDModelMixin
+
+
+class MorangoUUIDUtilsTestCase(TestCase):
+
+    def test_creating_same_instance_ID_model(self):
+        DatabaseIDModel.objects.create()
+        InstanceIDModel.get_or_create_current_instance()
+        firstIDModel = InstanceIDModel.objects.first()
+        (secondIDModel, _) = InstanceIDModel.get_or_create_current_instance()
+        self.assertEqual(firstIDModel, secondIDModel)
```

Comment:
:prepares-for-tests-failing-when-wifi-adapter-is-unplugged-at-just-the-wrong-moment:   :laughing: